src/devices/DCMotor/DCMotor.cs

```diff
@@ -0,0 +1,86 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Pwm;
+using System.Device.Pwm.Drivers;
+
+namespace Iot.Device.DCMotor
+{
+    public abstract class DCMotor : IDisposable
+    {
+        protected GpioController _controller;
+
+        protected DCMotor(GpioController controller)
+        {
+            _controller = controller ?? new GpioController();
+        }
+
+        public abstract double Speed { get; set; }
```

Comment:
Can we add summary comments to all these public facing API? Specifically it would be nice if the comments for properties like this one had the units so we know what we the value of this double means.

Reply:
Same comment about adding comment summaries to the public API applies to the rest of your classes, like the settings, etc.